Marlin/src/module/tool_change.cpp

```diff
@@ -251,10 +378,6 @@
       #endif
     }
 
-    #if HAS_HOTEND_OFFSET
-      current_position[Z_AXIS] += hotend_offset[Z_AXIS][active_extruder] - hotend_offset[Z_AXIS][tmp_extruder];
-    #endif
-
```

Comment:
Why is the normal solenoid-based `PARKING_EXTRUDER` code being altered in this PR?

Reply:
Same Issue as in Magnetic Parking Extruder. Z-Axis setup twice. also from caller method, because MPE was based from Parking Extruder. If wanted i'll revert this change.